recipe/meta.yaml

```diff
@@ -17,7 +17,7 @@ requirements:
     - bokeh
     - cloudpickle >=0.2.1
     - dask-core {{ version }}
-    - distributed >=1.21.0
+    - distributed >=1.21.1
```

Comment:
What causes this constraint? Also should it be in `setup.py`'s requirements for `dask`?

Reply:
It's not really a constraint, no more than the dask-core version number.  I figured this was typical for a metapackage to more strongly suggest version numbers

Reply:
If it's not a hard constraint, would it be ok for us to relax it?

Typically we try to mirror the requirements in `setup.py`. However things like extra requirements are added at people's discretion. The rules can be a bit looser with metapackages particularly if there is not a source of truth which it is tracking.

For context, was wanting to pin an older version of `distributed` so as to workaround issue ( https://github.com/dask/distributed/issues/1792 ) in the interim.